habinezadalvan/Banka
# #164959042 creates dashboard for new user

**what does this PR do?**
creates dashboard for new user
**Description of Task to be completed**
-create html content, css styles, and javascript functionality for login UI.
**How should this be tested manually**
-clone repo
-go in banka folder and open the index.html file with the browser will be able to access welcome page; then sign up and  to access first dashboard field.
-you can the create an account or see the transactions field
**relative pivotal tracker story**
#164959042
